src/Eshopworld.Messaging/ServiceBusFluentExtensions.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Eshopworld.Messaging
+{
+    using System;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Management.ServiceBus.Fluent;
+
+    /// <summary>
+    /// Contains extensions to the ServiceBus Fluent SDK: <see cref="Microsoft.Azure.Management.ServiceBus.Fluent"/>.
+    /// </summary>
+    public static class ServiceBusNamespaceExtensions
+    {
+        /// <summary>
+        /// Creates a specific queue if it doesn't exist in the target namespace.
+        /// </summary>
+        /// <param name="sbNamespace">The <see cref="IServiceBusNamespace"/> client that we are using to query entities inside the namespace.</param>
+        /// <param name="name">The name of the queue that we are looking for.</param>
+        /// <returns>The <see cref="IQueue"/> entity object that references the real queue.</returns>
+        public static async Task<IQueue> CreateQueueIfNotExists(this IServiceBusNamespace sbNamespace, string name)
```

Comment:
potentionally not needed with https://github.com/Azure/azure-service-bus-dotnet/pull/481 , let's check later as discussed

Reply:
The check/create will always be possible, just that the namespace manager makes the code easier to write. However most Azure resources are moving to RBAC away from connection strings, it's currently in preview for the ServiceBus, so moving to Fluent with MSI is a natural move for the package.